scripts/cluster_network.py

```diff
@@ -484,6 +484,13 @@ def update_bus_coordinates(
     else:
         n_clusters = int(snakemake.wildcards.clusters)
 
+    if params.base == "tyndp-raw":
+        # Fast-path if TYNDP base network is used
+        logging.info("Skipping clustering: not applicable for TYNDP base network")
+
+        busmap = n.buses.index.to_series()
+        linemap = n.lines.index.to_series()
+        clustering = pypsa.clustering.spatial.Clustering(n, busmap, linemap)
     if n_clusters == len(n.buses):
```

Comment:
I think this should be `elif`? Right now it works but mainly it is a coincidence since `tyndp-raw` is the same as `n_clusters == len(n.buses)` I believe.
```suggestion
    elif n_clusters == len(n.buses):
```